test/eth/TransactionManager.spec.js

```diff
@@ -158,14 +157,6 @@ describe('lifecycle hooks', () => {
     expect(lockHandlers.initialized).toBeCalledTimes(5);
     expect(lockHandlers.pending).toBeCalledTimes(5);
     expect(lockHandlers.mined).toBeCalledTimes(5);
-
-    log('\ndraw');
```

Comment:
why remove these?

Reply:
From what I can tell, they weren't actually being used for anything. It was timing out before it got through `wipe`, so I could put them back and increase the timeout on this test, but I think they might as well be removed if they're not testing anything.

Reply:
OK, fair enough :+1: